backend/twitter_bot.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import os
+import time
+import pprint
+import twitter
+import traceback
+import rethinkdb as r
+from secret import CONSUMER_KEY, CONSUMER_SECRET
+
+def respond(text):
+    print('Got one: %r' % text)
+    # TODO
+    return None
+
+def isAddressedToMe(tweet):
+    pprint.pprint(tweet)
+    return tweet['text'].lower().startswith('@' + me['screen_name'].lower())
+
+def on_tweet(tweet):
+    try:
+        _on_tweet(tweet)
+    except KeyboardInterrupt:
+        raise
+    except Exception as e:
+        traceback.print_exc()
+def _on_tweet(tweet):
+    if not isAddressedToMe(tweet):
+        return
+    response = respond(tweet['text'].split(' ', 1)[1])
+    if not response:
+        response = 'No response'
```

Comment:
Do not respond with "No response", it would make you spam everyone.
If we do not have response, we do not answer.
`return` is fine.